sdks/python/apache_beam/typehints/trivial_inference.py

```diff
@@ -303,10 +302,8 @@ def infer_return_type(c, input_types, debug=False, depth=5):
     elif inspect.isclass(c):
       if c in typehints.DISALLOWED_PRIMITIVE_TYPES:
         return {
-            list: typehints.List[Any],
-            set: typehints.Set[Any],
-            tuple: typehints.Tuple[Any, ...],
-            dict: typehints.Dict[Any, Any]
+            list: typehints.List[Any], set: typehints.Set[Any], tuple:
```

Comment:
This change makes things less readable IMO.

Reply:
The knob for this is each_dict_entry_on_separate_line I believe.

Reply:
Done.